scripts/deployment/artifactory.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+# To be executed from the root project directory.
+# Obtains R package from remote artifactory and executes local.sh script with package downloaded to temporary directory.
```

Comment:
I would remove all references to Artifactory in this code (comments, variable names etc.) since the code is more generic - it just retrieves files accessible through HTTP protocol. It doesn't matter if it's Artifactory or some other HTTP-supporting service. I would use descriptions like "repository" or "artifact repo" or something like this.